test/test_math_ops.cpp

```diff
@@ -319,246 +316,307 @@ TEST(MathOps, FloorModBroadcasting) {
   OpExecuter opexecuter(root, "FloorMod", static_input_indexes,
                         output_datatypes, sess_run_fetchoutputs);
 
+  opexecuter.RunTest();
+}  // end of test op FloorModBroadcasting
+
+// Test op: Log
+TEST(MathOps, Log1d) {
+  Scope root = Scope::NewRootScope();
+  int dim1 = 4;
+
+  Tensor A(DT_FLOAT, TensorShape({dim1}));
+
+  AssignInputValues(A, 1.4f);
+
+  vector<int> static_input_indexes = {};
+  auto R = ops::Log(root, A);
+
+  vector<DataType> output_datatypes = {DT_FLOAT};
+
+  std::vector<Output> sess_run_fetchoutputs = {R};
+  OpExecuter opexecuter(root, "Log", static_input_indexes, output_datatypes,
+                        sess_run_fetchoutputs);
+
   opexecuter.RunTest();
 }
 
-TEST(MathOps, Rsqrt) {
+TEST(MathOps, Log2d) {
   Scope root = Scope::NewRootScope();
   int dim1 = 2;
-  int dim2 = 2;
+  int dim2 = 3;
 
   Tensor A(DT_FLOAT, TensorShape({dim1, dim2}));
 
-  AssignInputValues(A, 4.0f);
+  AssignInputValues(A, 3.5f);
 
   vector<int> static_input_indexes = {};
-  auto R = ops::Rsqrt(root, A);
+  auto R = ops::Log(root, A);
 
   vector<DataType> output_datatypes = {DT_FLOAT};
 
   std::vector<Output> sess_run_fetchoutputs = {R};
-  OpExecuter opexecuter(root, "Rsqrt", static_input_indexes, output_datatypes,
+  OpExecuter opexecuter(root, "Log", static_input_indexes, output_datatypes,
                         sess_run_fetchoutputs);
 
   opexecuter.RunTest();
-}
+}  // end of test op Log
 
-TEST(MathOps, SquaredDifference) {
+// Test op: Minimum
+TEST(MathOps, Minimum) {
   Scope root = Scope::NewRootScope();
   int dim1 = 2;
   int dim2 = 2;
 
   Tensor A(DT_FLOAT, TensorShape({dim1, dim2}));
   Tensor B(DT_FLOAT, TensorShape({dim1, dim2}));
 
-  AssignInputValues(A, 7.5f);
-  AssignInputValues(B, 5.2f);
+  AssignInputValuesRandom(A);
+  AssignInputValuesRandom(B);
 
   vector<int> static_input_indexes = {};
-  auto R = ops::SquaredDifference(root, A, B);
+  auto R = ops::Minimum(root, A, B);
 
   vector<DataType> output_datatypes = {DT_FLOAT};
 
   std::vector<Output> sess_run_fetchoutputs = {R};
-  OpExecuter opexecuter(root, "SquaredDifference", static_input_indexes,
-                        output_datatypes, sess_run_fetchoutputs);
+  OpExecuter opexecuter(root, "Minimum", static_input_indexes, output_datatypes,
+                        sess_run_fetchoutputs);
 
```

Comment:
Minimum also supports broadcasting. Lets have a test for that mode too.

Reply:
Added